tox.ini

```
@@ -12,6 +13,8 @@ deps =
     django32: Django>=3.2,<3.3
     django40: Django>=4.0,<4.1
     django41: Django>=4.1,<4.2
+    django42: Django>=4.2,<4.3
+    django50: Django>=4.3,<5.1
```

I haven't tested this but I think this URL (or some variation of thereof) should work - basically the same pattern as `main`, but using `stable/5.0.x`.
```suggestion
    django50: django50: https://github.com/django/django/archive/stable/5.0.x.tar.gz
```

---

Updated :+1:

---

Thank you. Tests are failing, but that's on me - we've upgraded all repos to use ruff, and for some reason this has been left behind.